fl4health/preprocessing/autoencoders/vae_dim_reduction.py

```diff
@@ -0,0 +1,76 @@
+from abc import ABC
+from pathlib import Path
+
+import torch
+
+
+class Processing(ABC):
+    """
+    Abstact class to add processing methods.
+    """
+
+    def __init__(
+        self,
+        checkpointing_path: Path,
+    ) -> None:
+        self.checkpointing_path = checkpointing_path
+        self.DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
since DEVICE here is a class attribute rather than a constant, it should be lower-case (throughout the file)